lib/playground.dart

```diff
@@ -187,7 +191,12 @@ class Playground {
 
     keys.bind('ctrl-s', _handleSave);
     keys.bind('ctrl-enter', _handleRun);
-    keys.bind('f1', _handleHelp);
+    keys.bind('f1', () => _toggleDocTab());
+    document.onKeyUp.listen((e) => _handleHelp());
```

Comment:
It looks like this will send a documentation request per keystroke. That might be a bit excessive. Let's just ask for dartdoc on mouse clicks (as you have below).